stages.md

```diff
@@ -0,0 +1,49 @@
+# Stages for changes to GraphQL specification
+
+## Stage -2: proposed change(optional)
+**Prerequisite**: Described problem/change should be specific to the content of GraphQL Specification and not be an implementation detail.
+
+**Purpose**: Filter out question, issues for other repos and engage community discussion.
+
+## Stage -1: PR requested(optional)
+**Prerequisite**: Issue should contain description of problem/usecase and proposed solution.
+
+**Purpose**: Find member of community to be champion for this change.
+
+## Stage 0: Proposal
+**Prerequisite**:
++ Initial version of spec changes
++ Filled checklist in PR description
+
+**Steps**: start review process on specification changes
+
+**Post-Acceptance Changes Expected**: Major
+
+## Stage 1: Draft
+**Prerequisite**:
++ Finalized wording inside Specification document
++ Proposed spec changes don’t have any blind spots (not described edge-cases, missed changes to related part of spec, etc.)
+
+**Steps**: start working on “graphql-js” PR
```

Comment:
As this is the first mention of graphql-js in the document, should probably link to the repo.